### DISCUSSION TITLE
Max Width on GoAAppHeaderMenu popover

### DISCUSSION THREAD

**Discussion by chasebuhler** [GovAlta/ui-components#1302] [Q&A] [answered]
Is there a way to set the max width on the GoAAPpHeaderMenu popover like there is on the standalone popover component?

**Comment by ArakTaiRoth** [comment]
@chasebuhler Not at the moment there isn't. Are you needing to increase the width, or wanting to decrease the width? If you're wanting to increase the width, could you share with us the titles of the menu items that you're using?

**Reply by chasebuhler** [reply]
Hey Dustin, thanks for the reply. We are needing to increase it slightly. Our designers have coined the following:

![image](https://github.com/GovAlta/ui-components/assets/122830911/05f90684-4786-4557-9ee7-01706abc8093)

However, in practice, the first link has a line break:
![image](https://github.com/GovAlta/ui-components/assets/122830911/30080844-31da-4dcf-9f1b-1ad46e521bc5)

**Reply by chasebuhler** [reply]
@ArakTaiRoth Hey Dustin, is this something we should consult with our designers about instead?

**Reply by ArakTaiRoth** [reply]
@chasebuhler As I mentioned in my comment below. Your link seems unnecessarily verbose. Is this application already on alberta.ca? If so, that makes the `Alberta.ca` part unnecessary. Why is there a `for Business` part added? Is there another link somewhere that adds a link to a Personal account that is distinctly separate from this one?

**Reply by chasebuhler** [reply]
Thanks @ArakTaiRoth for the input. I will take this back to our designers for some discussion.

**Comment by ArakTaiRoth** [comment]
@twjeffery Any input on this? My thoughts personally, are decreasing the amount of text in the link, as that seems overly verbose.

**Reply by twjeffery** [reply]
Yea I agree with your recommendation Dustin. Although I recognize that we don't know the context, and content design is not only hard to do well, but often even harder to negotiate with the business on. 

But other than the content, can we allow teams to adjust the width? I'm sure that there are good edge cases (perhaps this included) that will have to be longer text. Currently the popover is set to a default width of 256px and a max width of 320px.

**Comment by ArakTaiRoth** [comment, marked as answer]
@chasebuhler Sorry for the extreme delay on this. But we've finally created a story that should help to alleviate your issue a little bit, you can see it and follow it here, #1335

**Reply by chasebuhler** [reply]
Thanks @ArakTaiRoth for the update. Apologies for a delay in response from our designers as well, as I have been away until today. Our designers have also adjust the designs to fit in the current dropdown. Appreciate the response on this.
